R/Filter.R

```diff
@@ -294,11 +294,8 @@ randomForestSRC.var.select = makeFilter( # nolint
     im = randomForestSRC::var.select(getTaskFormula(task), getTaskData(task),
       method = method, verbose = FALSE, always.use = getTaskFeatureNames(task),
       ...)
-    if (method == "md") {
-      setNames(im$varselect$depth, im$topvars)
-    } else if (method == "vh") {
-      im[["varselect"]][, 1L]
-    }
+	im$varselect[setdiff(rownames(im$varselect), im$topvars), 'depth'] = NA
+	im$topvars
```

Comment:
Is it safe that this change has no effect on `method = "vh"`? Just asking, because before we this dedicated split on how to deal with them differently

Reply:
Whether we are using method="vh" or method="md", the returned object always contains the components varselect and topvars. So it is safe to perform the operations above on the returned results for both methods. 
The component md.obj is only returned if method="md", but I have not used this.